components/03-modules/active-filters/active-filters.hbs

```diff
@@ -1,12 +1,14 @@
 <ul class="active-filters list {{class}}">
     {{# each filters }}
         <li class="active-filters__item list__item {{class}}">
-            <button type="button"
-                    class="active-filters__remove {{class}}"
-                    aria-label="{{ariaLabel}}"
+            <a
+                href="#"
+                class="active-filters__remove {{class}}"
+                aria-label="{{ariaLabel}}"
+                title="{{ariaLabel}}"
             >
                 {{ render '@icon' icon }}
-            </button>
+            </a>
             <span class="active-filters__label">
                 {{label}}&nbsp;:&nbsp;
```

Comment:
spaces `{{ label }}`

Reply:
`&nbsp;:&nbsp;` it looks a little weird, can we replace it?

Reply:
I left one &nbsp